inngest-spring-boot-demo/src/main/java/com/inngest/springbootdemo/DemoConfiguration.java

```diff
@@ -0,0 +1,82 @@
+package com.inngest.springbootdemo;
+
+
+import com.inngest.*;
+import com.inngest.springboot.InngestConfiguration;
+import kotlin.jvm.functions.Function2;
```

Comment:
we already discussed this but probably ideal if we don't have to pull in a kotlin Function here and can instead use lambdas (available in Java 8) or something like https://docs.oracle.com/javase/8/docs/api/java/lang/Runnable.html